applications/crossbar/priv/couchdb/schemas/system_config.callflow.json

```diff
@@ -64,14 +64,6 @@
             "description": "callflow post answer delay",
             "type": "integer"
         },
-        "privacy_name": {
```

Comment:
Something along the lines of https://github.com/2600hz/kazoo/blob/master/core/kazoo_media/src/kazoo_media_maintenance.erl#L22